### PR TITLE
[PWD-2516] Finalize Corporate Training Page

### PR DESCRIPTION
**Pull request by Vesals**
conditional deskripsi || ctDescription

minor css adjustments

sekalian fix import order warning di beberapa file 

### What type of PR is this? (_check all applicable_)

- [ ] Feature
- [ ] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [x] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### Screenshots (_optional_)

empty data: 
image

Muat lebih banyak tes limit 2 item:
image
image
image

Grid item, di < 1080 jadi xs = 6 biar chips ga ketimpa/ketutup:
image
image
image

### CODE REVIEW

File: src/components/ProgramCard.tsx [hunk 1]
```diff
@@ -127,7 +126,7 @@ const ProgramCard: FC<ProgramCardProps> = ({
           )}
         </Stack>
 
-        <ProgramDescription>{program.description}</ProgramDescription>
+        <ProgramDescription>{program.ctDescription}</ProgramDescription>
```
Comment (hiljaker) [inline src/components/ProgramCard.tsx:129]: ini buat conditional phyn `program.description || program.ctDecription` soalnya learning material datanya dr db

Comment (Vesals) [reply src/components/ProgramCard.tsx:129]: ooh oke nice zak

File: src/views/corporate-training/components/ListProgram.tsx [hunk 2]
```diff
@@ -33,6 +35,12 @@ const ListProgram: FC<ListProgramProps> = ({ state, selectedTab }) => {
     selectedLanguage
   ])
 
+  const cardToShow = useMemo<number>(() => {
```
Comment (hiljaker) [inline src/views/corporate-training/components/ListProgram.tsx:38]: namanya jgn cardToShow deh, soalnya hasil return disini grid size kan bukan jumlah cardnya?

Comment (Vesals) [reply src/views/corporate-training/components/ListProgram.tsx:38]: iyasi, jadi gridSide aja?

Comment (hiljaker) [reply src/views/corporate-training/components/ListProgram.tsx:38]: boleh tuh, boleh juga gridItemSize kalo mau lebih deskriptif wkwkwk atau cardSize

Comment (hiljaker) [reply src/views/corporate-training/components/ListProgram.tsx:38]: gua juga mau pake ini deh di learning material

**Review by kuntiarso — APPROVED** [review]
🔆🔆

**Review by hiljaker — APPROVED** [review]
sykadd

**Pull request merged**